doc/admin/administration.md

```diff
@@ -48,7 +50,9 @@ boro-11
 
 ```
 
-- Query Storage Device Health Data: `dmg storage query (device-health|target-health)`
+- Query Storage Device Health Data:
+  - `dmg storage query (device-health|target-health)`
+  - `dmg storage scan --nvme-health` shows NVMe controller health stats
```

Comment:
It may be helpful in both the metadata and health cases to explain the difference between the query command and the scan command, and the circumstances under which you'd use one rather than the other.

Reply:
I'm trying to get some consensus on that myself, when I do I will update the admin guide